voideditor/void
# Fix: Add Syntax Highlighting to Sidebar Code Snippets

# Resolve Code Highlighting Issue in Sidebar
#### Issue Reference: #76

---

### Description:

This PR resolves issue #76 where code snippets in the sidebar were not displaying syntax highlighting. The issue has been addressed, ensuring that code blocks in the sidebar are now properly highlighted, improving readability.

### How to Test:
- Open a sidebar that includes code snippets.
- Verify that the code is highlighted according to the selected programming language.


![image](https://github.com/user-attachments/assets/ea9e5742-f7ce-48c0-9afe-a0f0479f0c11)
